perllib/Open311/UpdatesBase.pm

```diff
@@ -188,6 +188,36 @@ sub _process_update {
         $request->{comment_time} = $p->whensent + DateTime::Duration->new( seconds => 1 );
     }
 
+    # Assign admin user to report if 'assigned_user_*' fields supplied
+    if ( $request->{extras} && $request->{extras}{assigned_user_email} ) {
+        my $assigned_user_email = $request->{extras}{assigned_user_email};
+        my $assigned_user_name  = $request->{extras}{assigned_user_name};
+
+        my $assigned_user
+            = FixMyStreet::DB->resultset('User')
+            ->search( { email => $assigned_user_email } )->first;
```

Comment:
Could this use `find()`? That is set up to make sure it only searches verified email addresses.